F sharp unit2

bobbychopra opened the pull request:
Finished updating F# bootcamp for Unit 2

skotzko commented on src/Unit-1/FSharp/lesson3/README.md:
```diff
@@ -0,0 +1,199 @@
+Compare your code to the solution in the [Completed](Completed/) folder to see what the instructors included in their samples.
+
+If everything is working as it should, the output you see should be identical to last time:
+![Petabridge Akka.NET Bootcamp Lesson 1.2 Correct Output](Images/working_lesson3.jpg)
```
this image is broken

skotzko commented:
@bobbychopra in general, this is looking great! Several images in these lessons are broken (e.g. lesson 1.3 readme) so I'd just step through the READMEs on GitHub to make sure the images are all good, and then I can merge it in. Great job! :+1:

bobbychopra commented:
Let me go through them again, thought I had fixed it

bobbychopra commented:
Sorry about the mess up ... I fixed them. Deleted PR#154, and changes are
in PR#151

On Mon, Oct 12, 2015 at 7:09 PM, Andrew Skotzko <notifications@github.com>
wrote:

> @bobbychopra <https://github.com/bobbychopra> in general, this is looking
> great! Several images in these lessons are broken (e.g. lesson 1.3 readme)
> so I'd just step through the READMEs on GitHub to make sure the images are
> all good, and then I can merge it in. Great job! [image: :+1:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/petabridge/akka-bootcamp/pull/154#issuecomment-147545569>
> .
>
